services/github/src/utils.js

```diff
@@ -173,6 +173,94 @@ export const tailLog = (log, maxLines = 250, maxPayloadLength = 30000) => {
     .join('\n');
 };
 
+/**
+ *
+ * @param {string} logString
+ * @param {number} maxPayloadLength
+ * @param {number} tailLines
+ * @returns string or null
+ */
+export const extractTailLinesFromLog = (logString, maxPayloadLength, tailLines) => {
+
+  if(tailLines === 0) {return null;}
```

Comment:
I think our eslint config is broken, it should have complained about the spaces and new lines 🙈 
```suggestion
  if (tailLines === 0) {
    return null;
  }
```

Reply:
i had ran yarn lint:fix, didnt show any complaints, will fix this.

Reply:
yeah, I don't know why .. maybe some files are being excluded. let me check

Reply:
or maybe I'm wrong and it was never part of the project's eslint.. heh.. nevermind then. Please ignore then :)

Reply:
I'll add a separate ticket